source/scss/stinson.scss

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ * Stinson
+ *
+ */
+@import 'shared';
+
+// mixin to extend stinson filter
+// @mixin stinson
+// @param $filters... {filter} - Zero to many css filters to be added
+// @example
+//   img {
+//     @include stinson;
+//   }
+//   or
+//   img {
+//     @include stinson(blur(2px));
+//   }
+//   or
+//   img {
+//     @include stinson(blur(2px)) {
+//       /*...*/
+//     };
+//   }
+@mixin stinson($filters...) {
+  @include filter-base;
+  filter: brightness(1.25) contrast(0.8) saturate(0.95) grayscale(0.05) sepia(0.05) $filters;
```

Comment:
Thanks for this PR! I did some adjustments, what do you think of using `    filter: brightness(1.15) contrast(.8) saturate(.85) sepia(.1);`

and altering the `::after` pseudo element to be:

```
background: rgba(240, 149, 128, .2);
mix-blend-mode: soft-light;
```

I think it looks a little bit closer (tiny bit lighter and warmer):
![screen shot 2016-11-02 at 11 05 37 pm](https://cloud.githubusercontent.com/assets/1693164/19954725/29e0b268-a151-11e6-92f9-2cd333ed3e20.jpg)